Jenkinsfile.deployment.mainnet

```diff
@@ -0,0 +1,45 @@
+@Library('lisk-jenkins') _
+pipeline {
+	agent none
+	triggers {
+		// Triggers every week at midnight on Sunday morning
+		cron('0 0 * * 0')
+		}
+	stages {
+		stage('Build docker images') {
+			agent any
+            when { branch 'development' }
+			steps {
+				sh 'make build-core'
+				sh 'make build-market'
+				sh 'make build-newsfeed'
+				sh 'make build-gateway'
+			}
+		}
+		stage('Deploy lisk-service Mainnet') {
+			agent any
+			when { branch 'development' }
+			environment {
+				ENABLE_HTTP_API='http-status,http-version2'
+				ENABLE_WS_API='blockchain,rpc-v2'
+				// Mainnet node URL
+				LISK_CORE_WS='ws://host.docker.internal:8080'
+				INDEX_N_BLOCKS='0'
+				ENABLE_JOB_MISSING_BLOCKS='true'
+				INDEX_MISSING_BLOCKS_RANGE='0'
+				ENABLE_TRANSACTION_STATS='true'
+				TRANSACTION_STATS_HISTORY_LENGTH_DAYS='366'
+			}
+			steps {
+				ansiColor('xterm') {
+					dir('./') {
+						sh 'make down'
+						sh 'make up-custom-core3-cd'
```

Comment:
I think this script is already removed. `make up` with right params should do the job.